kpatch-build/create-diff-object.c

```diff
@@ -4067,6 +4323,12 @@ int main(int argc, char *argv[])
 
 	kelf_orig = kpatch_elf_open(orig_obj);
 	kelf_patched = kpatch_elf_open(patched_obj);
+
+	kpatch_set_pfe_link(kelf_orig);
```

Comment:
I don't think it matters whether the original object had pfe.

Reply:
I'll look into this tomorrow...